addons/egoventure/cache/scene_cache.gd

```diff
@@ -29,6 +26,31 @@ var _queued_items: Array = []
 # Do not remove these scenes from the cache
 var _permanent_cache: PoolStringArray = []
 
+# Cache map
+var _cache_map: CacheMap
+
+# Cache sizes (in kilobyte) - current and maximum
```

Comment:
Please split the comment into two above every variable (background is that the API docs are generated from that)